src/plugins/dm.c

```diff
@@ -265,6 +265,27 @@ gchar* bd_dm_node_from_name (const gchar *map_name, GError **error) {
 }
 
 /**
+ * bd_dm_get_subsystem_from_name:
+ * @device_name: name of the device
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: subsystem of the given device
+ */
+gchar* bd_dm_get_subsystem_from_name (const gchar *device_name, GError **error) {
```

Comment:
Please add this function to ` docs/libblockdev-sections.txt` too.

Reply:
Fixed.